Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>    
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Excercise 1</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+  div{
+    padding:10px;
+  }
+</style>
+</head>
+<body>
+  <div>
+    <img src ="http://www.girldevelopitfargo.com/images/GDI.jpg">
+    <h1>About</h1>
+    <p>
+      It can be intimidating for women to learn and ask questions when they are in the <strong>extreme minority</strong>strong.
+      While open and welcoming, today's budding developer community is up to <strong>91% male</strong>. If we can empower more females with the confidence in their technological capabilities <em>we can begin to change this landscape</em>.
+    </p>
+    <h1>Our Locations</h1>
+    <ul>
+      <li>USA</li>
+      <ul>
+        <li><a href="#">New York</a></li>
+        <li><a href="#">Columbus</a></li>
+        <li><a href="#">Austin</a></li>
+        <li><a href="#">Philly</a></li>
+      </ul>
+      <li>World</li>
+      <ul>
+        <li><a href="#">Ottawa</a></li>
+        <li><a href="#">Sydney</a></li>
+      </ul>
+    </ul>
+
+    <h1>Upcoming Classes</h1>
+
+    <div class="container">
+      <div class="row">
+        <div class="col">Ney York</div>
+        <div class="col">JavaScipt & jQuery</div>
+        <div class="w-100"></div>
+        <div class="col">San Francisco</div>
+        <div class="col">HTML & CSS</div>
+      </div>
+    </div>
+
+    <h1>Sign Up!</h1>
+
+    <form>
+      Name: <input type="text" name="name"><br>
```

Comment:
Try to add more Bootstrap style to forms
https://getbootstrap.com/docs/4.1/components/forms/